packages/hot-module-replacement/client.js

```diff
@@ -50,7 +50,7 @@ let useHotCodePush = false;
 let forceReload = function () {
   useHotCodePush = true;
   // Wait until Reload package has been loaded
-  Meteor.startup(function () {
+  Meteor.startup(async function () {
```

Comment:
This file isn't compiled by ecmascript, so it can only use syntax that is supported by legacy browsers.

Reply:
We will don't make these changes to the client side.
So I will close this PR.

Thank you for your review.